src/cryptoadvance/specter/api/views.py

```diff
@@ -0,0 +1,14 @@
+"""
+Backend Template
+
+Use this to display server debugging info during development
+
+"""
+from flask import render_template
+from cryptoadvance.specter.api import api_bp
+
+
+@api_bp.route("/")
+def api():
+    """ rendering the documentation for the api """
+    return render_template("api.html", msg="API Blueprint View")
```

Comment:
i guess that was my code but i forgot to add an api.html? Can you maybe simply return the "API.md" ?
Should be rendered as simple text and it would be fine, imho.